service/lxd.go

```diff
@@ -200,7 +200,7 @@ func (s LXDService) Join(ctx context.Context, joinConfig JoinConfig) error {
 
 	err = op.WaitContext(ctx)
 	if err != nil {
-		return fmt.Errorf("Failed to configure cluster :%w", err)
+		return fmt.Errorf("Failed to configure cluster: %w", err)
```

Comment:
It seems to have turn up a `No matching cluster join operation found` error in https://github.com/canonical/microcloud/actions/runs/9611369723/job/26518376077?pr=323#step:13:58498

That's an error I also saw pop-up from time to time in the LXD test suite. Anyway, not relevant for this PR but would be nice to have that one addressed in LXD ;)

Reply:
Looks like it might be a race between creating the operation and checking for it.